server/middleware/classrooms.coffee

```diff
@@ -151,7 +162,7 @@ module.exports =
     code = req.body.code.toLowerCase()
     classroom = yield Classroom.findOne({code: code})
     if not classroom
-      throw new errors.NotFound(res) 
+      throw new errors.NotFound(res)
```

Comment:
The first argument of error object constructors is a human-readable error message string.